Sources/Library/Models/User.swift

```diff
@@ -52,11 +52,11 @@ public final class User: Model {
     /// Internal: Fluent::Model::init(Node, Context).
     public init(node: Node, in context: Context) throws {
         self.id = try? node.extract("id")
-        self.career_account = try node.extract("career_account")
-        self.first_name = try node.extract("first_name")
-        self.last_name = try node.extract("last_name")
+        self.career_account = try? node.extract("career_account")
```

Comment:
This is for when a PATCH request is made and we create a User object from the json body. Certain fields may or may not be present and we were previously getting 500 errors if the request didn't contain a first_name field, even though first_name was not being updated.